tabs/tab_advanced_stats.py

```
@@ -0,0 +1,133 @@
+                        - *VIF > 10*: High collinearity (consider removing variable).
+                    """),
+                    class_="p-3"
+                ),
+                col_widths=(6, 6)
+            ),
+            full_screen=True
+        )
+    )
+
+@module.server
+def advanced_stats_server(input, output, session, config: Any):
```

Ruff (0.14.10)

Unused function argument: `output`

(ARG001)